assets/js/components/ClustersList.jsx

```diff
@@ -71,7 +95,23 @@ function ClustersList() {
         filterFromParams: true,
         filter: (filter, key) => (element) =>
           element[key].some((sid) => filter.includes(sid)),
-        render: (_, { sid }) => sid.join(', '),
+        render: (_, { sid }) =>
+          sid.map((singleSid) => {
+            const linkData = getSapSystemLinkBySID(
+              applicationInstances,
+              databaseInstances,
+              singleSid
+            );
+
+            return (
+              <SapSystemLink
+                systemType={linkData ? linkData.type : null}
+                sapSystemId={linkData ? linkData.sap_system_id : null}
```

Comment:
You can use lodash's `get` function to get `type` and `sap_system_id` out of `linkData`